opendevin/core/main.py

```diff
@@ -40,6 +40,7 @@ async def run_agent_controller(
     sandbox: Sandbox | None = None,
     runtime_tools_config: dict | None = None,
     sid: str | None = None,
+    headless_mode: bool = False,
```

Comment:
Would it be a good idea to default this to `True`, so that we only need to change this to `False` in the backend without touching all over the places (e.g., tests, evaluations)?

Reply:
Yes that makes sense. When running OpenDevin via CLI, there's no way to "continue" once the application is paused.

Reply:
Also agree that this is probably OK.

Reply:
done in the latest commit to reduce the amount of changes in this PR